reporting/spacewalk-report

```diff
@@ -265,7 +265,7 @@ if __name__ == '__main__':
                             break
 
                 if outrow is not None:
-                    for m in report.multival_columns_reverted.keys():
+                    for m in list(report.multival_columns_reverted.keys()):
```

Comment:
If `multival_columns_reverted` is just a `dict`, then:

```python
for m in report.multival_columns_reverted:
```

Reply:
:+1: